### ISSUE TITLE
Research: Docker

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#1]
## User story
1. As a PO
2. I need skilled developers that know the tools they have to work with
3. So that productive work on the project can start as soon as possible

## Acceptance criteria
* Get an environment up and running in which you can run docker containers
* Do your research on [docker](https://docs.docker.com/get-started/overview/) and research how to use it if you are not already familiar
* Get familiar with the most common [docker commands](https://docs.docker.com/reference/)
* Get familiar with the syntax of [dockerfiles](https://docs.docker.com/engine/reference/builder/)
* Write a simple dockerfile and build a container from it. Mind dockerfile [best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
